shadows/framework/src/main/java/org/robolectric/shadows/ShadowParcelFileDescriptor.java

```diff
@@ -7,12 +7,14 @@
 
 import android.annotation.SuppressLint;
 import android.os.ParcelFileDescriptor;
+import com.ibm.icu.impl.Pair;
```

Comment:
@hoisie Do we recommend to use `Pair` in `ibm.icu` package? What about using `android.util.Pair` instead(https://stackoverflow.com/questions/40259381/android-util-pairf-s-example-in-android, https://developer.android.google.cn/reference/kotlin/android/util/Pair?hl=en)?

Reply:
Pair is not a good idea here, I had use a data structure `OpenedFileItem` instead.